### PR TITLE
Only versionize docs on new branches.

### PR DESCRIPTION
**Pull request by david-mcmahon**
Ref https://github.com/kubernetes/kubernetes/pull/27218 https://github.com/kubernetes/kubernetes/pull/27205

### CODE REVIEW

**Comment by lavalamp** [comment]
I don't know this script well enough. Can we make a test github repo (or just script that sets up a test git clone) and test this to see what it does? Ideally, test it continually?

**Comment by david-mcmahon** [comment]
@lavalamp yes, that's on the (in process) roadmap to get this running/mocked continuously (and tested).
I'll also file a specific issue to track.

**Comment by david-mcmahon** [comment]
ref #7

File: anago [hunk 1]
```diff
@@ -354,8 +354,8 @@ prepare_tree () {
     beta*|official) rev_version_base $label ;;
   esac
 
-  # versionize docs
-  if [[ "$PARENT_BRANCH" == master ]]; then
+  # versionize docs on new branches (from master) only
+  if [[ "$PARENT_BRANCH" == master && "$label" != "alpha" ]]; then
```
Comment (lavalamp) [inline anago]: I don't understand this, but can you just check if you're on the master branch? We should never run versionize docs on the master branch, I'm pretty sure.

Comment (david-mcmahon) [inline anago]: In this context, this condition is correct.  "alpha" releases only occur on the master.
But for readability, I'll update to condition specifically on the current branch.  Update coming..

**Comment by david-mcmahon** [comment]
PTAL

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**